router/rulerouter/rulerouter.go

```diff
@@ -256,12 +256,12 @@ func (r *RuleRouterImpl) PreRouteInitializedClientAdm(cl rclient.RouterClient) (
 		Msg("console client routed")
 
 	if err := cl.AssignRule(frRule); err != nil {
-		_ = cl.ReplyErrMsg("failed to assign rule")
+		_ = cl.ReplyErrMsg("failed to assign rule", "SPQRU")
 		return nil, err
 	}
 
 	if err := auth.AuthFrontend(cl, frRule); err != nil {
-		_ = cl.ReplyErrMsg(err.Error())
+		_ = cl.ReplyErr(err)
```

Comment:
This comment is not related to PR, but if we are working on mistakes, then my comment is still on the topic.

In the Go world, error variables [should start](https://go.dev/wiki/Errors#naming) with `Err' prefix. Many variable errors in our codebase that do not comply with this guide. It would be cool to rename them.

Reply:
Ok, I will do it in the next PR